### PR TITLE
RMHDR-256 Create Python and R Prototypes

### PR DESCRIPTION
**Pull request by pranavanba**
## Major Changes

1. Create prototypes in Python and R to do the following:
    1. Load parameters from a YAML config file
    2. Load a dataset and find the paths to only the participant partitions for which Participant Identifiers appear in a manifest file listing Participant Identifiers
    3. Build a filtered dataset using only the selected participant partition directory/file paths
2. Use a YAML config to store parameters

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+default:
+  default_key: default_value
+
+python:
+  participants_csv_path: "/sbgenomics/project-files/i2b2_exported_participant_ids_2024-01-18_142546_9276.csv"
```
Comment (itismeghasyam) [inline config.yml:5]: Do we need to change/update this path for every new archive version?

File: python/tests/test.py
Comment (itismeghasyam) [inline python/tests/test.py:1]: test.py is empty is this expected?

**Review by itismeghasyam — COMMENTED** [review]
Looks good. Except for some very minor issues. (see comments)

File: config.yml (hunk 1 above)
Comment (pranavanba) [reply config.yml:5]: Ideally, the path of this manifest file would remain unchanged, only the contents within (`Participant Identifier`, `cohort`) should change.

File: python/tests/test.py
Comment (pranavanba) [reply python/tests/test.py:1]: For now this is unnecessary with this being prototype code

**Pull request merged**